### PR TITLE
Add merge/test jobs for origin-web-console-server

### PR DESCRIPTION
**Pull request by jwforres**

### CODE REVIEW

**Comment by jwforres** [comment]
@stevekuznetsov does this seem reasonable for test/merge, anything obviously wrong structurally before i sync this up and start testing it?

FYI @spadgett

**Comment by jwforres** [comment]
aside from needing to build the generated files before i sync it

File: sjb/config/test_suites/test_branch_origin_web_console_server.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
```
Comment (stevekuznetsov) [inline sjb/config/test_suites/test_branch_origin_web_console_server.yml]: not necessary

File: sjb/config/test_suites/test_pull_request_origin_web_console_server.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
```
Comment (stevekuznetsov) [inline sjb/config/test_suites/test_pull_request_origin_web_console_server.yml]: not necessary

File: sjb/config/test_suites/merge_pull_request_origin_web_console_server.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+---
```
Comment (stevekuznetsov) [inline sjb/config/test_suites/merge_pull_request_origin_web_console_server.yml]: not necessary

File: sjb/config/test_cases/test_branch_origin_web_console_server_check.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+---
+parent: 'common/test_cases/origin.yml'
+overrides:
```
Comment (stevekuznetsov) [inline sjb/config/test_cases/test_branch_origin_web_console_server_check.yml]:
```yaml
sync_repos:
  - name: "whatever"
```

**Comment by jwforres** [comment]
Updated based on your feedback, generated dist, and pushed these jobs up to jenkins to test if they are working.  Just waiting for the repo to be available in the oct tool config in the AMI so I can test.

File: sjb/config/test_cases/test_branch_origin_web_console_server_check.yml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+      script: |-
+        # run commitchecker outside release container as it needs
+        # access to git; also explicitly force godeps verification
+        branch="$( git rev-parse --abbrev-ref --symbolic-full-name HEAD )"
+        if [[ "${branch}" == "master" ]]; then
+          RESTORE_AND_VERIFY_GODEPS=1 make verify-commits -j
+        fi
+    - type: "script"
+      title: "run check and verify tasks"
+      repository: "origin"
```
Comment (stevekuznetsov) [inline sjb/config/test_cases/test_branch_origin_web_console_server_check.yml]: seems wrong?

File: sjb/config/test_cases/test_branch_origin_web_console_server_check.yml [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+    - type: "script"
+      title: "verify commit history"
+      repository: "origin-web-console-server"
+      timeout: 7200
+      script: |-
+        # run commitchecker outside release container as it needs
+        # access to git; also explicitly force godeps verification
+        branch="$( git rev-parse --abbrev-ref --symbolic-full-name HEAD )"
+        if [[ "${branch}" == "master" ]]; then
+          RESTORE_AND_VERIFY_GODEPS=1 make verify-commits -j
```
Comment (stevekuznetsov) [inline sjb/config/test_cases/test_branch_origin_web_console_server_check.yml]: Does your repo need this?

**Comment by jwforres** [comment]
@stevekuznetsov updated. Looks like the AMI is still not updated to be able to sync the new repo? https://ci.openshift.redhat.com/jenkins/job/test_pull_request_origin_web_console_server_check/1/console

**Comment by stevekuznetsov** [comment]
Looks like an unrelated cascade of failures has meant we did not even preemptively tag any new AMIs for testing. I'll work through this and let you know when a new AMI is ready.

**Comment by jwforres** [comment]
Both tests are passing:
https://ci.openshift.redhat.com/jenkins/job/test_pull_request_origin_web_console_server_check/
https://ci.openshift.redhat.com/jenkins/job/test_pull_request_origin_web_console_server_e2e/

**Pull request merged**